cloud_run_service_connect_cloud_sql/main.tf

```diff
@@ -0,0 +1,188 @@
+# Project data
+data "google_project" "project" {
+}
+
+# Enable Secret Manager API
+resource "google_project_service" "secretmanager_api" {
+  service            = "secretmanager.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Enable SQL Admin API
+resource "google_project_service" "sqladmin_api" {
+  service            = "sqladmin.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Enable Cloud Run API
+resource "google_project_service" "cloudrun_api" {
+  service            = "run.googleapis.com"
+  disable_on_destroy = false
+}
+
+# Creates SQL instance (~15 minutes to fully spin up)
```

Comment:
Some comments are verb based, some aren't. Make consistent?

# Enable
# Create

and so on